lib/ansible_tower_client/api.rb

```diff
@@ -1,4 +1,5 @@
 require 'pp'
+require 'faraday'
```

Comment:
I would put a comment here:
`# We use Faraday exception class constants in method_missing below`

Or, just put the `require 'faraday'` in `method_missing` below since it's the only place (I think) that uses the `Faraday` constant.